lib.php

```diff
@@ -1984,7 +1986,7 @@ function facetoface_user_cancel($session, $userid=false, $forcecancel=false, &$e
  * @returns string Error message (or empty string if successful)
  */
 function facetoface_send_notice($postsubject, $posttext, $posttextmgrheading,
-                                $notificationtype, $facetoface, $session, $userid) {
+                                $notificationtype, $facetoface, $session, $userid,$htmlmessage) {
```

Comment:
Need to update the PHP docs for this function to include the new variable $htmlmessage.
Also `$userid,$htmlmessage` needs to be this with a space `$userid, $htmlmessage`

Reply:
Fixed.